render-thumb-for.sh

```diff
@@ -36,9 +36,8 @@ if [ -n "${XDG_CONFIG_HOME}" ]; then
 fi
 
 if [ -n "$is_cmd_kitten" ]; then
-    if [ ! "$(kitten icat --detect-support 2>&1)" ]; then
```

Comment:
I think you meant:
```bash
if ! kitten icat --detect-support >/dev/null 2>&1; then
```

Reply:
maybe I'm doing something wrong but this condition never evaluates as true, even from kitty
```bash
if ! kitten icat --detect-support >/dev/null 2>&1; then
    is_cmd_kitten_icat_support=true
fi
```

this expression shows "memory" in the in the shell, printed by the --detect-support command, but does evaluate true in kitty and false in iTerm2
```bash
if kitten icat --detect-support 2>&1; then
    is_cmd_kitten_icat_support=true
fi
```

this expression does not show "memory" and evaulates true in kitty, falce in iTerm2
```bash
if [ $(kitten icat --detect-support 2>&1) ]; then
    is_cmd_kitten_icat_support=true
fi
```